src/pointer-analysis/dynamic_object_name.cpp

```diff
@@ -0,0 +1,34 @@
+/*******************************************************************\
+
+Module: Dynamic object name
+
+Author: Marius-Constantin Melemciuc
+
+Date: April 2017
+
+@ Copyright Diffblue, Ltd.
+
+\*******************************************************************/
+
+#include "dynamic_object_name.h"
+
+/*******************************************************************\
+
+Function: get_dynamic_object_prefix
+
+  Inputs:
+
+ Outputs: The prefix of the dynamic-object,
+          composed of "value_set::dynamic_object".
+
+ Purpose: To generate a prefix for dynamic-objects suitable for use
+          in the LHS of value-set maps.
+
+\*******************************************************************/
+
+const std::string &get_dynamic_object_prefix()
+{
+  static std::string prefix="value_set::dynamic_object";
```

Comment:
It was requested in another PR by this comment [1]. 

[1] https://github.com/diffblue/security-scanner/pull/85#discussion_r110210174